Documentation/user-manual.txt

```diff
@@ -1065,10 +1077,24 @@ available, as in dialogue *B*, above. Currently these are:
  - Poseidon MK VI eCCR dive logs
  - CSV (text-based and spreadsheet-based) dive logs, including APD CCR logs
 
-Selecting the appropriate format and then the specific log file in the large window
-containing the file list on the right of the dialogue, opens
-the imported dive log in the _Subsurface_ *Dive List*. Some other formats not
-accessible through the Import dialogue are also supported, as explained below.
+Selecting the appropriate format from the dropdown list and then the specific log file in the main window
+containing the file list. Then select the _Open_ button near the bottom right. This opens a dialog
+containing the intermediate CSV representation of the dive data (see image below).
+
+image::images/Import_csv.jpg["FIGURE: Import dialogue: step 2",align="center"]
+
+Normally the user will ignore this dialogue and immediately select the _OK_ button near the bottom right.
+In this case ignore the following paragraph.
```

Comment:
This is true for a few dive computers/log software. Others will require manually choosing the correct columns.

Reply:
text changed